integration-tests/common/log.go

```diff
@@ -0,0 +1,16 @@
+package common
+
+import "encoding/json"
+
+type LogResponse struct {
+	Status string    `json:"status"`
+	Data   []LogData `json:"data"`
+}
+
+type LogData struct {
+	Filename string `json:"filename"`
```

Comment:
Should we go as far as checking the log line contents? 🤔 
I think that would be okay, but I wouldn't go any further than that.